frontend/src/components/v1/InlineAlertTray.css

```
@@ -0,0 +1,127 @@
+
+.inline-alert-tray__icon {
+  flex-shrink: 0;
+  display: flex;
+  align-items: flex-start;
+  padding-top: 0.125rem;
+}
+
+.inline-alert-tray__content {
+  flex: 1;
+  color: var(--text-main, #ffffff);
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)